docs/source/bubbleColumn.rst

```diff
@@ -1,12 +1,302 @@
+
+.. code-block:: console
+
+   cd ${BCE_CASE}/
+   surfaceToPatch -tol 1e-3 inlets.stl
+   export newmeshdir=$(foamListTimes -latestTime)
+   rm -rf constant/polyMesh/
+   cp -r $newmeshdir/polyMesh ./constant
+   rm -rf $newmeshdir
+   cp ${BCE_CASE}/constant/polyMesh/boundary /tmp
+   sed -i -e 's/inlets\.stl/inlet/g' /tmp/boundary
+   cat /tmp/boundary > constant/polyMesh/boundary
+ 
+
+Initial conditions
+------------
+The initial conditions are defined through the ``${BCE_CASE}/0/`` time folder. We provide a premade folder ``${BCE_CASE}/0.orig/``. Two fields, the volume fraction of gas and liquid are essential in a bubble column and are left for the user to define. We typically define them with the ``setFields`` utility in OpenFOAM which looks up the values defined in ``${BCE_CASE}/system/setFieldsDict``. The important part of the file is shown below
+
+.. code-block:: txt
+
+   defaultFieldValues
+   (
+       volScalarFieldValue alpha.gas 0.99
+       volScalarFieldValue alpha.liquid 0.01
+   );
+   
+   regions
+   (
+       boxToCell
+       {
+           box (-1.0 -1.0 -1.0) (10 7 10);
+           fieldValues
+           (
+               volScalarFieldValue alpha.gas 0.01
+               volScalarFieldValue alpha.liquid 0.99
+           );
+       }
+   );
+
+Here, everything below y=7m is (almost) pure liquid and the rest (``defaultFieldValues``) is (almost) pure gas. 
+
+In the end, the initial conditions are defined by running
+
+.. code-block:: console
+
+   cd ${BCE_CASE}
+   cp -r 0.orig 0
+   setFields -dict system/setFieldsDict
+
+Mesh post-treatment 2
+------------
+At this point, the reactor can still be post treated. We can, for example, ensure that the liquid volume is 20L by doing 
+
+.. code-block:: console
+
+   transformPoints "scale=(0.19145161188225573 0.19145161188225573 0.19145161188225573)"
+
+
+Global variables
+------------
+Several cases in BiRD adjust the boundary conditions according to the ``${BCE_CASE}/constant/globalVars`` file. This is useful to get a holistic view of the whole case setup. In this case, ``globalVars`` defines ``uGasPhase`` which is used as an inlet boundary condition in ``${BCE_CASE}/0.orig/U.gas``.
+
+In practice, the gas velocity is set using a ``VVM`` which can result in different gas velocity depending on the size of the inlet or the size of the reactor. This is shown in ``globalVars_temp`` and ``globalVars`` as  ``uGasPhase #calc "$liqVol * $VVM / (60 * $inletA * $alphaGas)";``.
+Crucially, one needs to set ``liqVol`` (total volume of liquid in m3) and ``inletA`` (inlet area in m2) correctly. This can be done using a mix of OpenFOAM utilities (to get the ``inletA`` value) and BiRD utilities (to get the ``liqVol`` value). The script ``writeGlobalVars.py`` reads the erroneous ``globalVars_temp`` and writes the correct ``globalVars`` with the appropriate ``liqVol`` and ``inletA``.
+
```

Comment:
also add a line about gas composition, is it N2/O2? if so, i am guessing only O2 is mass transferred from gas to liquid. get_kla script can be used calculate kla

Reply:
OK I'll add a note that this is how we set the gas comp as well.
We don't have get_kla in that case. I'd prefer to have the post process scripts in the bird python package (`bird/postprocess`). So I won't add the note about the get_kla script, is that ok?

Reply:
sounds good!